### ISSUE TITLE
Not seeing aria labels on day buttons

### ISSUE THREAD

**Issue by iantbarker** [gpbl/react-day-picker#1546]
### Discussed in https://github.com/gpbl/react-day-picker/discussions/1545

<div type='discussions-op-text'>

<sup>Originally posted by **iantbarker** September  7, 2022</sup>
On the docs examples, the days buttons have an aria-label with the date on it, but on the newest versions, it doesn't have that label. Am I missing a setting to enable that or was this removed in some version? I tried different versions in a codesandbox and couldn't find one that includes the labels. It would really helpful to have those labels for jest testing purposes.

This is from a documentation instance:
<img width="517" alt="Screen Shot 2022-09-07 at 1 19 31 PM" src="https://user-images.githubusercontent.com/7671200/188950238-05aa721f-0f55-46ee-a7cf-b86f90522b44.png">

This is what is on 8.2:
<img width="523" alt="Screen Shot 2022-09-07 at 1 20 41 PM" src="https://user-images.githubusercontent.com/7671200/188950435-b6f14d33-43b1-44dd-beac-a4717e3de2f0.png">
</div>
